.travis.yml

```diff
@@ -50,7 +50,15 @@ before_install:
       export PHPUNIT_X="$PHPUNIT --exclude-group tty,benchmark,intl-data"
       export COMPOSER_UP='composer update --no-progress --no-suggest --ansi'
 
-      nanoseconds() {
+      if [[ $PHP = 5.* ]]; then
+          composer () {
+              $HOME/.phpenv/versions/$TRAVIS_PHP_VERSION/bin/composer config platform.php $TRAVIS_PHP_VERSION
```

Comment:
this should be the exact PHP version. `$TRAVIS_PHP_VERSION` will make it use `5.5` AFAICT, which means `5.5.0`. this might cause issues with packages having `5.5.9` as min version.